CONTRIBUTING.md

```diff
@@ -70,20 +71,73 @@ When you are contributing to changelog, please follow these suggestions:
 
 3. Once you have completed your work, create a Pull Request, ensuring that it meets the requirements listed below.
 
-### Testing
+### Requirements for Pull Requests
+
+* Please create Pull Requests against the `master` branch.
+* Please make sure that your code complies with [PEP8](https://www.python.org/dev/peps/pep-0008/).
+* One line should not contain more than 100 characters.
+* Make sure that new code is covered by a test case (new or existing one).
+* We don't like [spaghetti code](https://en.wikipedia.org/wiki/Spaghetti_code).
+* All the tests in the test-suite have to pass.
+
+## Development Environment
+
+### Requirements
+
+Currently, the development environment can be set up on __Linux__, and __MacOS__.
```

Comment:
Are you sure?

As far as I know, `make` can be used on Windows using MinGW or `Chocolatey` as well. Docker is also available on Windows.

Reply:
@Aniket-Pradhan yes I'm aware of that, but I still didn't add Windows because I was not sure if it would work. I'm not sure if the docker commands stay the same. I could still add Windows in the docs, but would like to confirm if all the instructions work the same way as in Linux.

Reply:
There's no difference between Docker for Windows and Docker for Linux/Unix in commands. So, the commands stay the same.

I also ran release-bot and tests on my Windows computer, which works just fine.

Reply:
@Aniket-Pradhan Great, I was just talking to @TomasTomecek a few days ago to check it on windows!

Reply:
as I said earlier, none of us use Windows, so we won't be able to provide any support there